compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -226,6 +225,24 @@ pub struct InliningMap<'tcx> {
     inlines: GrowableBitSet<usize>,
 }
 
+struct MonoItems<'tcx> {
```

Comment:
Could you document this struct and its fields?

Reply:
Judging inlining does seem a little out of place here. I added the `extend` method to the MonoItems so it looks more reasonable

Reply:
sure